tiledb/sm/serialization/tiledb-rest.capnp.c++

```diff
@@ -5,24 +5,23 @@
 
 namespace capnp {
 namespace schemas {
-static const ::capnp::_::AlignedData<209> b_ce5904e6f9410cec = {
+static const ::capnp::_::AlignedData<208> b_ce5904e6f9410cec = {
```

Comment:
why does the size get smaller when you add a field?

Reply:
I don't know

Reply:
This is auto-generated, hopefully not an issue.